src/components/Menu.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Disclosure, Transition } from '@headlessui/react'
+
+
+const Menu = (): JSX.Element =>  {
```

Comment:
I also recommend adding comments above your React components explaining what it is/does. Though it's pretty obvious from this component's name what it does, it can be good practice to include a description of components (throwback to Java docs)